scripts_staging/Win_Hello_World.ps1

```diff
@@ -6,6 +6,8 @@
     Hello World example for a PowerShell script in TRMM.
     $log if provided will output verbose logs with timestamps. This can be used to determine how long the installer took.
 
+    This is a test of the pull request system. This is just a test.
```

Comment:
A test is only effective if it tests something :)

Reply:
Now I see it!!!

Reply:
Tracking down the mystery....this was step 4